tests/CMakeLists.txt

```diff
@@ -168,7 +168,8 @@ if(MFEM_MGIS_HAVE_TFEL)
   endfunction(add_micromorphic_damage_2d_test)
 
   if(MFEM_USE_SUITESPARSE)
-	add_micromorphic_damage_2d_test(2)
+          message(STATUS "micromorphic_damage_2d has been disabled with SUITE SPARSE")
+          #add_micromorphic_damage_2d_test(2)
```

Comment:
```suggestion
```

Reply:
I'll soon be reactivating this test, so I'd rather keep it as a comment. We'll post a tag once this test and the next two will be reactivated.

Reply:
In progress (Branch #87 )